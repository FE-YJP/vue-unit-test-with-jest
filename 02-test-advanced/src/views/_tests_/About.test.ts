import { shallowMount } from '@vue/test-utils'
import About from '../About.vue'

describe('views.About', () => {
  describe('snapshot', () => {
    it('snapshot测试', () => {
      const wrapper = shallowMount(About)
      expect(wrapper.vm.$el).toMatchSnapshot()
    })
  })
})
