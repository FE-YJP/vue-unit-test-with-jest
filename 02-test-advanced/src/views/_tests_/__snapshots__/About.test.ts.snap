// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views.About snapshot snapshot测试 1`] = `
<div
  class="about"
>
  <h1>
    This is an about page
  </h1>
</div>
`;
